.travis.yml

```diff
@@ -1,17 +1,23 @@
 cache: bundler
 sudo: false
 rvm:
-  - 2.2
-  - 2.3
   - 2.4
+  - 2.5
+  - 2.6
+  - 2.7
+  - ruby-head
 gemfile:
-  - gemfiles/rails3.2.gemfile
   - gemfiles/rails4.2.gemfile
   - gemfiles/rails5.0.gemfile
   - gemfiles/rails5.1.gemfile
   - gemfiles/rails5.2.gemfile
+  - gemfiles/rails6.0.gemfile
 script: "bundle exec rake test"
 matrix:
   exclude:
     - rvm: 2.4
-      gemfile: gemfiles/rails3.2.gemfile
```

Comment:
Let’s remove this gemfile from the repository, too.

Reply:
👍 removed in 42e4dfe6e0eca2bea48207d2d1361ccf92b2662f